starport/services/network/network.go

```diff
@@ -117,17 +117,17 @@ func InitializationHomePath(homePath string) InitOption {
 	}
 }
 
-// InitializationKeyringBackend provides the keyring backend to use to initialize the blockchain
-func InitializationKeyringBackend(keyringBackend chaincmd.KeyringBackend) InitOption {
+// InitializationGenesisURL provides a genesis url for the initial genesis of the chain blockchain
```

Comment:
```suggestion
// InitializationGenesisURL provides a genesis URL for the initial genesis of the chain blockchain
```

Reply:
I know it's only comments, but we had an inconsistency of case, so let's be consistent and correct w/regard to URL